src/Compilers/CSharp/Test/Semantic/Diagnostics/IOperationTests.cs

```diff
@@ -55,5 +55,39 @@ static void Test2(int y, params int[] x)
   IArgument (Matching Parameter: x) (OperationKind.Argument)
     ILiteralExpression (Text: null) (OperationKind.LiteralExpression, Type: null, Constant: null)");
         }
+
+        [Fact]
+        public void DeconstructionAssignmentFromTuple()
+        {
+            var text = @"
+public class C
+{
+    public static void M()
+    {
+        int x, y, z;
+        (x, y, z) = (1, 2, 3);
+        (x, y, z) = new C();
+        var (a, b) = (1, 2);
+    }
+    public void Deconstruct(out int a, out int b, out int c)
+    {
+        a = b = c = 1;
+    }
+}";
+            var compilation = CreateCompilationWithMscorlib(text, references: new[] { ValueTupleRef, SystemRuntimeFacadeRef }, parseOptions: TestOptions.RegularWithIOperationFeature);
+            compilation.VerifyDiagnostics();
+            var tree = compilation.SyntaxTrees.Single();
+            var model = compilation.GetSemanticModel(tree);
+
+            var assignments = tree.GetRoot().DescendantNodes().OfType<AssignmentExpressionSyntax>().ToArray();
+            Assert.Equal("(x, y, z) = (1, 2, 3)", assignments[0].ToString());
+            Assert.Equal(null, model.GetOperation(assignments[0]));
```

Comment:
I didn't think about that. Thanks.
I can think of two mitigations: let the assignment operation be returned but neuter is (Kind = None, etc), or go back to a using a new bound node.
I've made a commit with the first option. The trouble is that an interface/type check still reveals it is an assignment :-(
We can discuss tomorrow and decide whether that is sufficient.